app/api/route/message_api.rb

```diff
@@ -6,6 +6,8 @@ class MessageApi < Base
       optional :thread_id, type: Integer, desc: 'ID of thread'
       optional :order_by, type: Symbol, values:%i( created_at id  ), desc: 'Order of returned issues.'
       optional :order, type: Symbol, values:%i(asc desc), default: :asc
+      optional :id_lte, type: Integer, desc: 'Issue ID lower or equal than'
+      optional :id_gte, type: Integer, desc: 'Issue ID greater or equal than'
```

Comment:
I'd prefer this done by date, have a `after` and `before` with a date then do this with `created_at`.  What do you think?  Have you already written code that expects `id_lte`?

Reply:
No, I can easily change it to `after_id` and `before_id`.